AEPServices/Sources/ui/MessageMonitor.swift

```diff
@@ -0,0 +1,45 @@
+/*
+ Copyright 2020 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+import Foundation
+
+///This class is used to monitor if an UI message is displayed at some point in time,
+///currently this applies for full screen and alert messages.
+///The status will be exposed to the core through the UIService.
+///@see UIService.isMessageDisplayed()
+public class MessageMonitor {
```

Comment:
all of the classes and protocols are public. we should be considering what access level each one needs

Reply:
we also need to consider which ones need to be exposed to obj-c w/ `@objc`